js/index.js

```diff
@@ -1,7 +1,78 @@
 // Iteration 1: Names and Input
-
+let hacker1 = "Phillip";
+console.log(`The driver's name is ${hacker1}`);
+let hacker2 = "Clara";
+console.log(`The navigator's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+  } else if (hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+  } else {
+    console.log(`Wow, you both have equally long names, ${hacker2.length} characters!.`);
+  }
+
+// Iteration 3: Loops
+let nameWithSpaces1 = "";
+
+for (let i=0; i < hacker1.length; i++){
+  nameWithSpaces1 += hacker1[i] + " ";
+}
+
+console.log(nameWithSpaces1.slice(0,13).toUpperCase());
+
+let nameWithSpaces2 = "";
+
+for (let j=hacker2.length-1; j >= 0; j--) {
+  nameWithSpaces2 += hacker2[j];
+}
+
+console.log(nameWithSpaces2);
+
+if (hacker2[0] < hacker1[0]) {
```

Comment:
How about: Andrè and Alfonso? These two names will both trigger the console.log("What?! You both have the same name?")

Instead you can solve this with localeCompare() to prevent this bug